common_factories.py

```diff
@@ -274,7 +298,7 @@ def addWinTests(
                 "dojob",
                 '"',
                 util.Interpolate(
-                    f'"C:\Program Files (x86)\Microsoft Visual Studio\\2022\BuildTools\Common7\Tools\VsDevCmd.bat" -arch=%(kw:arch)s && cd mysql-test && {cmd}',
+                    f'"C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\Common7\\Tools\\VsDevCmd.bat" -arch=%(kw:arch)s && cd mysql-test && {cmd}',
```

Comment:
@cvicentiu 
Why do you think `\` needs to be escaped?
`Buildbot-worker `is already handling string interpolation on Windows.

Look for example on line 0 in https://buildbot.mariadb.org/#/builders/234/builds/35330/steps/11/logs/stdio
`dojob ^"\^"^" ^"\^"C:\Program Files ^(x86^)\Microsoft Visual Studio\2022\BuildTools\Common7\Tools\VsDevCmd.bat\^" -arch=x64 ^&^& cd mysql-test ^&^& ..\mysql-test\collections\buildbot_suites.bat ^&^& cd ..^" ^"\^"^"
`